nap/fields.py

```diff
@@ -153,6 +161,26 @@ def descrub_value(self, val, for_read=False):
         return obj_list
 
 
+class SimpleListField(Field):
+    """A list of simple types (e.g. strings, not Resources)"""
+    def scrub_value(self, val):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
+
+    def descrub_value(self, val, for_read=False):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
```

Comment:
I like the way it's currently written. I think we should change falsey objects like `''`, `0`, and `False` to `['']` `[0]`, and `[False]`.